apps/storybook/src/Dialog.stories.tsx

```diff
@@ -277,3 +277,64 @@ export const DraggableRelativeToContainer: Story = ({ isOpen, ...rest }) => {
     </div>
   );
 };
+
+export const Position: Story = () => {
```

Comment:
story should be renamed to `Placement` too.

after that, update the visual test too.

Reply:
got it

Reply:
old image (`Dialog.test.ts-Position.png`) can be deleted